web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -5,27 +5,40 @@ import 'rxjs/add/operator/timeout';
 
 import { StoreService } from 'app/store/store.service';
 import { Settings } from 'app/settings';
-import { Account } from '@burstjs/core';
+import { Account, compose, ApiSettings } from '@burstjs/core';
 import { generateMasterKeys, Keys, encryptAES, hashSHA256, getAccountIdFromPublicKey } from '@burstjs/crypto';
 import { isBurstAddress, convertNumericIdToAddress, convertAddressToNumericId } from '@burstjs/util';
+import { environment } from 'environments/environment.prod';
 
 
 @Injectable()
 export class AccountService {
     private nodeUrl: string;
+    private api: any; //todo
 
     public currentAccount: BehaviorSubject<any> = new BehaviorSubject(undefined);
 
     constructor(private storeService: StoreService) {
         this.storeService.settings.subscribe((settings: Settings) => {
             this.nodeUrl = settings.node;
         });
+
+        const apiSettings = new ApiSettings(environment.defaultNode, 'burst');
+        this.api = compose(apiSettings);
```

Comment:
ApiSettings moved here @ohager

Reply:
Perfect 👌, eventually we can centralize in environment files